admin/src/Models/Span.js

```diff
@@ -68,12 +68,11 @@ export const mergePeriods = toMergePeriods => {
 
 // Filter not deleted spans for one category, return sorted.
 export const filterCategory = (spans, category) =>
-    spans.filter(i => !i.deleted_at &&  i.type === category).sort((a, b) => a.startdate > b.startdate);
+    spans.filter(i => !i.deleted_at &&  i.type === category).sort();
 
 
 // Return assembled periods for non deleted spans in a category.
 export const filterPeriods = (spans, category) => {
-    spans.sort((a, b) => a.start > b.start ? 1 : -1);
```

Comment:
We also sort by periods there: https://github.com/makerspace/makeradmin/blob/f8f401bc373b5e1a86528398b0456b3db0503d09/admin/src/Models/CategoryPeriods.js#L36

This function should probably be changed to reflect that it is working on periods not spans? I don't remember this code  too well, so can be wrong.

Reply:
This sort is also just overridden on the next line as spans are sorted again in filterCategory()?